server/core/src/main/scala/org/infinispan/server/core/transport/NettyChannelInitializer.scala

```diff
@@ -15,12 +18,14 @@ import io.netty.handler.ssl.SslHandler
  * @author Galder Zamarreño
  * @since 4.1
  */
-class NettyChannelInitializer(server: ProtocolServer,
-                                  encoder: ChannelOutboundHandler)
+class NettyChannelInitializer(server: ProtocolServer, transport: => NettyTransport,
+                                       encoder: ChannelOutboundHandler)
       extends ChannelInitializer[Channel] {
 
    override def initChannel(ch: Channel): Unit = {
       val pipeline = ch.pipeline
+//      pipeline.addLast("netty-logging", new LoggingHandler(LogLevel.ERROR))
```

Comment:
Just a heads up - there will be a clash with my PR here: https://github.com/infinispan/infinispan/pull/4247